notebooks/models/text-to-text/T5/master_processing.ipynb

```diff
@@ -0,0 +1,530 @@
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "melted_df"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-08-03T12:29:53.324505Z",
+     "start_time": "2023-08-03T12:29:53.318773Z"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "outputs": [],
+   "source": [
+    "melted_df.rename(columns={'Marked for processing':'MFP','Value':'question'},inplace=True)"
```

Comment:
You can rename this columns when you use the melt function

Reply:
done